app/components/Transactions/index.js

```diff
@@ -287,4 +327,33 @@ function debounce(func, timeout = 300){
     clearTimeout(timer);
     timer = setTimeout(() => { func.apply(this, args); }, timeout);
   };
+}
+
+function isNegativeValue(type) {
+  switch (type) {
+    // Positive
+    case 'RECEIVE':
+    case 'COINBASE':
+    case 'REDEEM':
+    case 'REVEAL':
+    case 'REGISTER':
+      return false;
+
+    // Neutral
+    case 'UPDATE':
+    case 'RENEW':
+    case 'OPEN':
+    case 'FINALIZE':
+    case 'CLAIM':
+      return false;
+
+    // Negative
+    case 'SEND':
+    case 'BID':
+      return true;
+
+    // Should not reach here
+    default:
+      return false;
+  }
```

Comment:
```js
return ['SEND', 'BID'].includes(type);
```

Reply:
True, this is much simpler! And can keep pos/neg/neutral as comments in short.
This is actually the same function that's used in `Transaction` (singular) component, that can be update too.